version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "1.11"
\ No newline at end of file
+ThisBuild / version := "1.2"
```

Comment:
I don't know what's convention we're using. Should it be 1.20? 
Otherwise it looks fine.

Reply:
@bakennedy ^^^

Reply:
Also, @bakennedy we need to update the readme with the new version as well.

Reply:
Good catch @keyur9